### ISSUE TITLE
Why does it act like this? The color of data-toggle, data-target and aria-expanded is not the same as for class and id in the button tag. its in black color. how to change this

### ISSUE THREAD

**Issue by suneera** [MaxCDN/bootstrap-cdn#852]

**Comment by XhmikosR** [comment]
Sorry, this isn't support for Bootstrap; you should ask on their repo for help.

**Comment by suneera** [comment]
then how can i use this data toggle in notpad++
